tests/functional/database-management.test.js

```diff
@@ -176,3 +326,19 @@ function noDbExpectedFailureTests() {
         });
     });
 };
+
+
+function calculateTotalSize(keyValueSize, valueSize) {
+    return keyValueSize + valueSize;
+}
+
+function calculateKeySize(key) {
```

Comment:
This function is doing a very confusing job of either calculating if string, or parroting back the original value if a number is  given, thus not really doing what the function is named as. This can be simplified with the mentioned `createKeys` change.

Reply:
Removed